### ISSUE TITLE
[BUG] Unable to create Alert

### ISSUE THREAD

**Issue by Rodionc** [TheHive-Project/TheHive4py#207]
### Request Type
Bug 

### Work Environment

| Question              | Answer
|---------------------------|--------------------
| OS version (server)       | Ubuntu 20.04.1 LTS
| OS version (client)       | Ubuntu 20.04.1 LTS
| TheHive4py version / git hash   |  1.8.0

### Problem Description
Unable to create Alert from Samples/test-alert-create.py. Running fail with python error in models.py

### Steps to Reproduce
1. Add APIKey to test-alert-create.py
2. Run  *python3 test-alert-create.py* -> Got error  
3. Remove 1.8.0 vesrion: *pip3 uninstall thehive4py*
4. Install 1.7.1 version: *pip3 install thehive4py==1.7.1*
5. Run  *python3 test-alert-create.py* -> Alert created successfully 

### Possible Solutions
Temporary solution: downgrade thehive4py

## Error:
Traceback (most recent call last):
  File "test-alert-create.py", line 20, in <module>
    AlertArtifact(dataType='file', data='pic.png'),
  File "/home/user/.local/lib/python3.8/site-packages/thehive4py/models.py", line 697, in __init__
    file_object = open(data[0], 'rb')
FileNotFoundError: [Errno 2] No such file or directory: 'p'

**Comment by nadouani** [comment]
Hello @Rodionc thanks for this issue. In fact it looks like a regression related to alerts with file artifacts.

I'll fix it in a 1.8.1 patch release

**Comment by nadouani** [comment]
Fixed this issue to make sure the code below is correct:

For Alert Artifacts

```
inmemory_file = open("hello.txt", "rb")
artifacts = [
    AlertArtifact(dataType='file', data='sample.txt'),
    AlertArtifact(dataType='file', data=(inmemory_file, 'hello.txt'))
]
```

For Case Observables

```
CaseObservable(dataType='file', data='sample.txt', tags=['thehive4py'])

inmemory_file = open("hello.txt", "rb")
CaseObservable(dataType='file', data=(inmemory_file, 'hello.txt'), tags=['thehive4py'])
```
